### PR TITLE
Fix code styling bug

### PR DESCRIPTION
**Pull request by etiennelandais**

### CODE REVIEW

File: src/Common/AbstractProcessor.php [hunk 1]
```diff
@@ -88,9 +85,8 @@ public function handleException(Exception $exception, $level = 'warning', array
         $title = 'Handle Exception: '. $level;
         $messageException = $exception->getMessage();
         $message = $this->formatNotification->formatMessage($title,false,$messageException);
```
Comment (etiennelandais) [inline src/Common/AbstractProcessor.php]: MINOR Put exactly one space after comma separated arguments. [rule](http://localhost:9000/coding_rules#rule_key=php%3AS1808)

File: src/Common/AbstractProcessor.php [hunk 2]
```diff
@@ -88,9 +85,8 @@ public function handleException(Exception $exception, $level = 'warning', array
...
-        $this->logger->$level(
-            $message, $context
-        );
+        $this->logger->$level($message, $context);
+
```
Comment (etiennelandais) [inline src/Common/AbstractProcessor.php]: MINOR Remove the useless trailing whitespaces at the end of this line. [rule](http://localhost:9000/coding_rules#rule_key=php%3AS1131)

File: src/Common/AbstractProcessor.php [hunk 3]
```diff
@@ -40,10 +40,7 @@
      * @param EventDispatcherInterface $dispatcher
      * @param LoggerInterface          $logger
      */
-    public function __construct(
-        EventDispatcherInterface $dispatcher,
-        LoggerInterface $logger
-    ) {
+    public function __construct(EventDispatcherInterface $dispatcher, LoggerInterface $logger) {
```
Comment (etiennelandais) [inline src/Common/AbstractProcessor.php]: MINOR Move this open curly brace to the beginning of the next line. [rule](http://localhost:9000/coding_rules#rule_key=php%3AS1808)

File: tests/Stub/Entity/Operation.php [hunk 4]
```diff
@@ -213,12 +213,13 @@ public function setWithdrawnAmount($withdrawnAmount)
     {
         $this->withdrawnAmount = $withdrawnAmount;
     }
-    function getOriginAmount()
+
```
Comment (etiennelandais) [inline tests/Stub/Entity/Operation.php]: MINOR Remove the useless trailing whitespaces at the end of this line. [rule](http://localhost:9000/coding_rules#rule_key=php%3AS1131)

**Comment by etiennelandais** [comment]
SonarQube analysis reported 10 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 9 minor

Watch the comments in this conversation to review them.

#### 6 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [Initializer.php#L280](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c08aea025a3e8df6c94fd5065faea021308fe24f/src/Cashout/Initializer.php#L280): Split this 129 characters long line (which is greater than 120 authorized). [rule](http://localhost:9000/coding_rules#rule_key=php%3AS103)
1. ![MINOR][MINOR] [HiPay.php](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c08aea025a3e8df6c94fd5065faea021308fe24f/src/Api/HiPay.php): Add a new line at the end of this file. [rule](http://localhost:9000/coding_rules#rule_key=php%3AS113)
1. ![MINOR][MINOR] [HiPay.php#L531](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c08aea025a3e8df6c94fd5065faea021308fe24f/src/Api/HiPay.php#L531): Replace this "else if" keyword sequence by "elseif" keyword. [rule](http://localhost:9000/coding_rules#rule_key=php%3AS1793)
1. ![MINOR][MINOR] [HiPay.php#L632](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c08aea025a3e8df6c94fd5065faea021308fe24f/src/Api/HiPay.php#L632): Replace this "else if" keyword sequence by "elseif" keyword. [rule](http://localhost:9000/coding_rules#rule_key=php%3AS1793)
1. ![MINOR][MINOR] [LogOperations.php](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c08aea025a3e8df6c94fd5065faea021308fe24f/tests/Stub/Entity/LogOperations.php): Add a new line at the end of this file. [rule](http://localhost:9000/coding_rules#rule_key=php%3AS113)
1. ![MINOR][MINOR] [Operation.php](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c08aea025a3e8df6c94fd5065faea021308fe24f/tests/Stub/Entity/Operation.php): Add a new line at the end of this file. [rule](http://localhost:9000/coding_rules#rule_key=php%3AS113)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by etiennelandais** [comment]
SonarQube analysis reported 54 issues
* ![CRITICAL][CRITICAL] 27 critical
* ![MAJOR][MAJOR] 14 major
* ![MINOR][MINOR] 13 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [HiPay.php#L139](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c92df5d3854e88adf9b8827b1bf532b0f5d3d8aa/src/Api/HiPay.php#L139): Define a constant instead of duplicating this literal "request.options/headers/php-auth-user" 12 times. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1192)
1. ![CRITICAL][CRITICAL] [HiPay.php#L141](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c92df5d3854e88adf9b8827b1bf532b0f5d3d8aa/src/Api/HiPay.php#L141): Define a constant instead of duplicating this literal "request.options/headers/php-auth-pw" 12 times. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1192)
1. ![CRITICAL][CRITICAL] [HiPay.php#L145](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c92df5d3854e88adf9b8827b1bf532b0f5d3d8aa/src/Api/HiPay.php#L145): Define a constant instead of duplicating this literal "request.options/headers/php-auth-subaccount-id" 10 times. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1192)
1. ![CRITICAL][CRITICAL] [HiPay.php#L326](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c92df5d3854e88adf9b8827b1bf532b0f5d3d8aa/src/Api/HiPay.php#L326): Define a constant instead of duplicating this literal "locale" 3 times. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1192)
1. ![CRITICAL][CRITICAL] [HiPay.php#L326](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c92df5d3854e88adf9b8827b1bf532b0f5d3d8aa/src/Api/HiPay.php#L326): Define a constant instead of duplicating this literal "en_GB" 3 times. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1192)
1. ![CRITICAL][CRITICAL] [HiPay.php#L494](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c92df5d3854e88adf9b8827b1bf532b0f5d3d8aa/src/Api/HiPay.php#L494): Define a constant instead of duplicating this literal "GetUserAccount" 4 times. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1192)
1. ![CRITICAL][CRITICAL] [HiPay.php#L533](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c92df5d3854e88adf9b8827b1bf532b0f5d3d8aa/src/Api/HiPay.php#L533): Define a constant instead of duplicating this literal "request.options/headers/php-auth-subaccount-login" 4 times. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1192)
1. ![CRITICAL][CRITICAL] [UserAccount.php#L187](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c92df5d3854e88adf9b8827b1bf532b0f5d3d8aa/src/Api/HiPay/Model/Rest/UserAccount.php#L187): Define a constant instead of duplicating this literal "contact_informations" 11 times. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1192)
1. ![CRITICAL][CRITICAL] [BankInfo.php#L25](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c92df5d3854e88adf9b8827b1bf532b0f5d3d8aa/src/Api/HiPay/Model/Status/BankInfo.php#L25): Add a "case default" clause to this "switch" statement. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS131)
1. ![CRITICAL][CRITICAL] [Mirakl.php#L80](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c92df5d3854e88adf9b8827b1bf532b0f5d3d8aa/src/Api/Mirakl.php#L80): Define a constant instead of duplicating this literal "request.options/headers/Authorization" 6 times. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/Common/AbstractProcessor.php (hunk 1 above)
Comment (etiennelandais) [inline src/Common/AbstractProcessor.php]: MINOR Put exactly one space after comma separated arguments. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)

File: src/Common/AbstractProcessor.php (hunk 2 above)
Comment (etiennelandais) [inline src/Common/AbstractProcessor.php]: MINOR Remove the useless trailing whitespaces at the end of this line. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1131)

File: src/Common/AbstractProcessor.php (hunk 3 above)
Comment (etiennelandais) [inline src/Common/AbstractProcessor.php]: MINOR Move this open curly brace to the beginning of the next line. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)

**Comment by etiennelandais** [comment]
SonarQube analysis reported 7 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 6 minor

Watch the comments in this conversation to review them.

#### 4 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [Initializer.php#L280](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c92df5d3854e88adf9b8827b1bf532b0f5d3d8aa/src/Cashout/Initializer.php#L280): Split this 129 characters long line (which is greater than 120 authorized). [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS103)
1. ![MINOR][MINOR] [HiPay.php#](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c92df5d3854e88adf9b8827b1bf532b0f5d3d8aa/src/Api/HiPay.php#): Add a new line at the end of this file. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS113)
1. ![MINOR][MINOR] [HiPay.php#L531](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c92df5d3854e88adf9b8827b1bf532b0f5d3d8aa/src/Api/HiPay.php#L531): Replace this "else if" keyword sequence by "elseif" keyword. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1793)
1. ![MINOR][MINOR] [HiPay.php#L632](https://github.com/hipay/hipay-wallet-cashout-mirakl-library/blob/c92df5d3854e88adf9b8827b1bf532b0f5d3d8aa/src/Api/HiPay.php#L632): Replace this "else if" keyword sequence by "elseif" keyword. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1793)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/Common/AbstractProcessor.php [hunk 5]
```diff
@@ -88,9 +86,8 @@ public function handleException(Exception $exception, $level = 'warning', array
         $title = 'Handle Exception: '. $level;
         $messageException = $exception->getMessage();
         $message = $this->formatNotification->formatMessage($title,false,$messageException);
```
Comment (etiennelandais) [inline src/Common/AbstractProcessor.php]: MINOR Put exactly one space after comma separated arguments. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1808)

File: src/Common/AbstractProcessor.php [hunk 6]
```diff
@@ -88,9 +86,8 @@ public function handleException(Exception $exception, $level = 'warning', array
...
-        $this->logger->$level(
-            $message, $context
-        );
+        $this->logger->$level($message, $context);
+
```
Comment (etiennelandais) [inline src/Common/AbstractProcessor.php]: MINOR Remove the useless trailing whitespaces at the end of this line. [rule](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1131)

**Review by jprotin — APPROVED** [review]

**Pull request merged**